cmd/calyptia/delete_pipeline_cluster_objects.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func newCmdDeletePipelineClusterObjects(config *config) *cobra.Command {
```

Comment:
This method is actually deleting a single cluster object.

```suggestion
func newCmdDeletePipelineClusterObject(config *config) *cobra.Command {
```